tableauserverclient/server/request_options.py

```diff
@@ -112,13 +112,13 @@ def max_age(self):
         return self._max_age
 
     @max_age.setter
-    @property_is_int(range=(0, 240))
+    @property_is_int(range=(0, 240), allowed=[-1])
```

Comment:
I added `allowed` for cases where the allowed value is non-contiguous. Like a range of 1-100, and -1. Here you could just use a range of -1 to 240.

That said, this sort of makes it more readable, that -1 is a 'different' kind of value, so I think it's fine either way

Reply:
That's a really good point. I will keep it since it does make it more readable